backend/api/tests/test_views.py

```
@@ -0,0 +1,113 @@
+@pytest.fixture
+def authenticated_api_client(api_client, test_user):
+    """Fixture for an authenticated API client."""
+    # Note: In a real scenario with JWT/Firebase, you'd mock the authentication
+    # or use a mechanism to force authenticate. For basic DRF session auth (if enabled)
+    # or token auth, you might use force_authenticate.
+    # Since we use Firebase, proper testing requires mocking firebase_admin.auth
+    # or setting up test tokens. For now, we'll simulate by forcing authentication.
+    api_client.force_authenticate(user=test_user)
+    return api_client
+
+def test_auth_test_view_authenticated(authenticated_api_client):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_auth_test_view_authenticated`

Add return type annotation: `None`

(ANN201)